### PR TITLE
Remove mention of lazy loading images from comparison page

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
## Proposed Changes

* Remove mention of lazy loading images from comparison page
* Remove something called a tour about lazy loading images. I cannot recall seeing this before and I could not track down a place it is currently being used, but removed it anyway

## Testing Instructions

1. Use the Calypso live link or checkout this branch locally
2. Go to `/features/comparison` on jetpack cloud and make sure the Lazy Loading row is gone
image

These are the only instances I could find on Calypso, you could also check to see if I missed any. I searched for "Lazy Loading", "Lazy loading" and "lazy loading". 

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/layout/guided-tours/config.js [hunk 1]
```diff
@@ -12,7 +11,6 @@ import mediaBasicsTour from 'calypso/layout/guided-tours/tours/media-basics-tour
 export default [
 	checklistSiteTitle,
 	jetpackChecklist,
-	jetpackLazyImages,
```
Comment (jboland88) [inline client/layout/guided-tours/config.js:15]: I found where this tour is used. For a Jetpack site that has a purchase (I tested with security), Click on "Upgrades" in the menu and then "My Plan" from the pricing view.

On the my plan page, there is a checklist of Jepack items that start these guided tours of settings in Calypso.
Screenshot 2023-09-20 at 11 17 00 AM

**Review by jboland88 — CHANGES_REQUESTED** [review]
I found a couple more instances searching just "lazy load".

One was the checklist task in `current-plan/jetpack-checklist/index.tsx` and the other was the actual lazy load setting in calypso in `speed-up-site-settings.jsx`.

I'm not sure if we want to remove the actual feature toggle, since the description on it was recently changed: https://github.com/Automattic/jetpack/pull/33030

I'd default to leaving the feature toggle for now and cleaning up the instance of the tour in the Jetpack checklist 👍

**Comment by CodeyGuyDylan** [comment]
> I'd default to leaving the feature toggle for now and cleaning up the instance of the tour in the Jetpack checklist 👍

Thank you for finding this, I defaulted to leaving functionality in because I believe a different team is going to be removing that later on. That is why I don't have a diff related to the plugin for this 😄

**Review by jboland88 — APPROVED** [review]
LGTM! 🚢

**Pull request merged**